lib/routes/mofish/index.js

```diff
@@ -18,12 +19,17 @@ module.exports = async (ctx) => {
     ctx.state.data = {
         title,
         link: `https://mo.fish/`,
-        description: title,
-        item: data.map((item) => ({
-            title: item.Title,
-            pubDate: new Date(item.releaseTime).toUTCString(),
-            link: item.Url,
-            guid: item.id,
-        })),
+        item: data.map((item) => {
+            const isImage = Number(id) === 136;
+            const description = isImage ? `<p><img src="${item.Url}" referrerpolicy="no-referrer"><br></p>` : title;
```

Comment:
Please use art-template for HTML element. See https://github.com/DIYgod/RSSHub/discussions/8359
docs: https://docs.rsshub.app/joinus/script-standard.html#template

No need to add `referrerpolicy="no-referrer"`, RSSHub will add this automatically